ruth/data/map.py

```diff
@@ -56,10 +57,18 @@ def __init__(self, border, data_dir="./data", with_speeds=True):
 
         if fresh_data:
             self._store()
-        self.hdf5_file_path = str((Path(data_dir) / "map.hdf5").absolute())
-        self.osm_to_hdf_map_ids = self.to_hdf5(self.hdf5_file_path)
+
+        self.save_hdf()
+
+    def save_hdf(self):
+        self.hdf5_file_path = str((Path(self.data_dir) / "map.hdf5").absolute())
```

Comment:
This doesn't change after the class is created, so `self.hdf5_file_path` should be set in the constructor.